src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/GetElevationAtAPoint/GetElevationAtAPoint.xaml.cs

```diff
@@ -0,0 +1,107 @@
+        private readonly Uri _elevationUri = new Uri("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer");
+
+        // Starting point of the observer.
+        private readonly MapPoint _observerPoint = new MapPoint(83.9, 28.42, SpatialReferences.Wgs84);
+
+        // Create the graphics overlay.d
+        private GraphicsOverlay _overlay;
+
+        // Create symbols for the text and marker.
+        private SimpleMarkerSceneSymbol _elevationMarker;
+        private TextSymbol _elevationTextSymbol;
+        private readonly Graphic _elevationTextGraphic = new Graphic();
+
+        public GetElevationAtAPoint()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization.
+            Initialize();
+
+            // Handle taps on the scene view for getting elevation.
+            MySceneView.GeoViewTapped += SceneViewTappedAsync;
+        }
+
+        private void Initialize()
+        {
+            // Create the camera for the scene.
+            Camera camera = new Camera(_observerPoint, 30000.0, 10.0, 80.0, 0.0);
+
+            // Create scene.
+            Scene myScene = new Scene(Basemap.CreateImageryWithLabels())
+            {
+                // Set the initial viewpoint.
+                InitialViewpoint = new Viewpoint(_observerPoint, 1000000, camera)
+            };
+
+            // Create the marker for showing where the user taps.
+            _elevationMarker = SimpleMarkerSceneSymbol.CreateCylinder(Color.Red, 10, 750);
+
+            // Create the text for displaying the elevation value.
+            _elevationTextSymbol = new TextSymbol("", Color.Red, 20, Esri.ArcGISRuntime.Symbology.HorizontalAlignment.Center, Esri.ArcGISRuntime.Symbology.VerticalAlignment.Middle);
+            _elevationTextGraphic.Symbol = _elevationTextSymbol;
+
+            // Add base surface for elevation data.
+            myScene.BaseSurface.ElevationSources.Add(new ArcGISTiledElevationSource(_elevationUri));
+
+            // Create the graphics overlay.
+            _overlay = new GraphicsOverlay
+            {
+                SceneProperties = new LayerSceneProperties(SurfacePlacement.Absolute)
+            };
+            MySceneView.GraphicsOverlays.Add(_overlay);
+
+            // Add the scene to the view.
+            MySceneView.Scene = myScene;
+        }
+
+        private async void SceneViewTappedAsync(object sender, Esri.ArcGISRuntime.UI.Controls.GeoViewInputEventArgs e)
+        {
+            // Check that the point is on the surface.
+            if (e.Location != null)
+            {
+                // Clear any existing graphics from the graphics overlay.
+                _overlay.Graphics.Clear();
+
+                // Get the elevation value.
+                double elevation = await MySceneView.Scene.BaseSurface.GetElevationAsync(e.Location);
```

Comment:
Because the surface is the highlight of the sample, I'd suggest storing the surface as a field and referring to it directly, rather than going through the scene view. That will make it more clear that surfaces can be used independent of visualization.